### DISCUSSION TITLE
Kubepps with Pinniped Documentation

### DISCUSSION THREAD

**Discussion by jfcoronado** [kubeapps/kubeapps#3260] [Q&A] [answered]
Good Day, 

I'm trying to configure Kubeapps with Pinniped for OIDC Authentication, and the link to the documentation: [https://kubeapps.com/docs/using-an-OIDC-provider/using-an-OIDC-provider-with-pinniped](url) send me 404 Not Found. 

https://kubeapps.com/docs/using-an-OIDC-provider/

**Comment by jfcoronado** [comment, marked as answer]
I found the righ document :-) https://github.com/kubeapps/kubeapps/blob/master/docs/user/using-an-OIDC-provider-with-pinniped.md

**Reply by antgamdia** [reply]
¡Hola! Unfortunately, we have noticed a bunch of broken links in our online docs (eg: https://github.com/kubeapps/kubeapps/issues/3180). This is inherent to how the docs are being generated -it requires a manual upgrade- . However, we have short-term plans to change the way we generate the docs so that we don't have any broken links anymore.

By the way, if you need some assistance while setting up Kubeapps with Pinniped, do not hesitate to contact us via discussions or even using the [#Kubeapps channel in Slack](https://kubernetes.slack.com/messages/kubeapps).
Also, we recently wrote a more detailed guide on how to install Kubeapps in Tanzu Kubernetes Grid using also Pinniped, some steps could also be useful for you: https://github.com/kubeapps/kubeapps/tree/master/docs/step-by-step/kubeapps-on-tkg
